File: Web/RMA-Web-Shell/projects/fincare/src/app/billing-manager/models/adhoc-payment-instruction.ts
export class AdhocPaymentInstruction {
  adhocPaymentInstructionId: number;
  dateToPay: Date;
  adhocPaymentInstructionStatus: number;
  reason: string;
  rolePlayerId: number;
  rolePlayerName: string;
  amount: number;
  createdBy: string;
  modifiedBy: string;
  createdDate: Date;
  modifiedDate: Date;
  finPayeNumber: string;
  errorDescription: string;
  bankAccountEffectiveDate: Date;
  bankAccountNumber: string;
  bankAccountType: string;
  bankAccountHolder: string;
  bankBranchCode: string;
  bank: string;
  policyId?: number;
  policyNumber: string;
  targetedTermArrangementScheduleIds: number[];
  rolePlayerBankingId: number;
  tempDocumentKeyValue: string;
}

File: Service/Billing/RMA.Service.Billing.Contracts/Entities/AdhocPaymentInstruction.cs
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

using System.Collections.Generic;

namespace RMA.Service.Billing.Contracts.Entities
{
    public class AdhocPaymentInstruction
    {
        public int AdhocPaymentInstructionId { get; set; } // AdhocPaymentInstructionId (Primary key)
        public System.DateTime DateToPay { get; set; } // DateToPay
        public AdhocPaymentInstructionStatusEnum AdhocPaymentInstructionStatus { get; set; } // AdhocPaymentInstructionStatusId
        public string Reason { get; set; } // Reason (length: 50)
        public decimal Amount { get; set; } // Amount
        public int RolePlayerId { get; set; } // RolePlayerId
        public string RolePlayerName { get; set; } // RolePlayerName (length: 255)
        public System.DateTime CreatedDate { get; set; } // CreatedDate
        public System.DateTime ModifiedDate { get; set; } // ModifiedDate
        public string CreatedBy { get; set; } // CreatedBy (length: 50)
        public string ModifiedBy { get; set; } // ModifiedBy (length: 50)
        public string FinPayeNumber { get; set; }
        public string ErrorDescription { get; set; }
        public System.DateTime BankAccountEffectiveDate { get; set; }
        public string BankAccountNumber { get; set; }
        public string BankAccountType { get; set; }
        public string BankAccountHolder { get; set; }
        public string BankBranchCode { get; set; }
        public string Bank { get; set; }
        public bool IsInitiatedByDebitOrderGenerationProcess { get; set; }
        public int? PolicyId { get; set; }
        public string PolicyNumber { get; set; }
        public List<int> TargetedTermArrangementScheduleIds { get; set; }
        public int? RolePlayerBankingId { get; set; } // RolePlayerBankingId
        public string TempDocumentKeyValue { get; set; }
    }
}
